Library/Homebrew/extend/ARGV.rb

```diff
@@ -25,7 +25,7 @@ def formulae
   def resolved_formulae
     require "formula"
     @resolved_formulae ||= (downcased_unique_named - casks).map do |name|
-      if name.include?("/")
+      if name.include?("/") || File.exist?(name)
```

Comment:
Sorry, after re-reading the commit message I think I understand this better. But I think the check should be a bit stricter, e.g. something like:

```ruby
(File.file?(name) && name.end_with?(".rb"))
```

And could you please explain what issue this solves, as I'm still unsure about this?